packages/rolldown/src/builtin-plugin/constructors.ts

```diff
@@ -77,3 +78,9 @@ export function viteResolvePlugin(
   })
   return makeBuiltinPluginCallable(builtinPlugin)
 }
+
+export function moduleFederationPlugin(
+  config: BindingModuleFederationPluginOption,
+): BuiltinPlugin {
+  return new BuiltinPlugin('builtin:load-fallback', config)
```

Comment:
```suggestion
  return new BuiltinPlugin('builtin:load-fallback', config)
```
Is `builtin:load-fallback` right?

Reply:
Fixed it.